spec/views/hyrax/admin/admin_sets/show.html.erb_spec.rb

```diff
@@ -0,0 +1,42 @@
+describe "hyrax/admin/admin_sets/show.html.erb", type: :view do
+  let(:admin_set) { build(:admin_set, id: '123', title: ['Example Admin Set'], creator: ['jdoe@example.com']) }
+  let(:work) { build(:work, title: ['Example Work Title'], admin_set: '123') }
```

Comment:
Good eye. This seems to correspond with my broken test (sigh). I'll get this fixed up today, along with the other fixes you've suggested.